projecteuler/helpers/number_checks.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+def is_palindromic_number(number):
+    if not isinstance(number, int):
+        raise TypeError("Input must be an integer")
+
+    number_string = str(number)
+    number_of_digits = len(number_string)
+    if number_of_digits <= 1:
+        return True
+
+    halfway_digit = int(number_of_digits / 2)
+    for count, char in enumerate(number_string):
```

Comment:
a simple range with subscripts would be better here:

```python
for index in range(0, number_of_digits // 2):
    end_index = 0 - index - 1
    if number_string[index] != number_string[end_index]:
        return False

return True
```

where `//` is "integer divide"

Reply:
Ah I hadn't seen the `//` syntax - useful. I took a brief look and then tried rewriting so I wasn't copying direct.